### DISCUSSION TITLE
Storage postgres

### DISCUSSION THREAD

**Discussion by ALtairJoker** [vercel/community#2996] [Help]
### Summary

The storage fills up too fast and I have not used more than 10 MB of data

### Example

storage

### Steps to Reproduce

a

**Comment by da-stoi** [comment]
Same issue here. I have under 20 rows across all tables. There are already two threads about this open: #2990 #2981
<img width="325" alt="image" src="https://github.com/vercel/community/assets/20031472/47320e72-10cd-4f1f-8b99-d96df3a93b75">

**Comment by amyegan** [comment]
We had a display bug for storage size reporting which has been resolved. You should no longer see this issue, so please let me know if it is still happening for you 🙇
